packages/fxa-auth-server/package.json

```diff
@@ -19,7 +20,7 @@
     "stop": "pm2 stop pm2.config.js",
     "restart": "pm2 restart pm2.config.js",
     "test": "VERIFIER_VERSION=0 scripts/test-local.sh",
-    "test-ci": "npm run gen-keys && scripts/test-local.sh && npm run test-e2e",
```

Comment:
I noticed this in passing. gen-keys is run in test-local.sh so this is redundant